lib/article_status_manager.rb

```diff
@@ -110,6 +109,10 @@ def update_title_and_namespace(synced_articles)
                         deleted: false)
         # Find corresponding Assignment records and update the titles
         AssignmentUpdater.update_assignments_for_article(article)
+
```

Comment:
Please test this locally. I don't think it will be working as written, as it does not pass the article to the manager.

Also, it will need to compare the old namespace and the new namespace (from `article` vs `article_data`), because this path also handles cases where only the title but not the namespace changed, or where the namespace changed but it was not in mainspace either before or after.

Reply:
Uhm so ideally an old namespace belonging to the article and a new namespace after the update?
Also the build is failing due to a rubocop lint on method name which I wonder why it started failing just now but the method name hasn't changed.

Reply:
Also how do I move an article page from mainspace to a diferent namespace on the app? So far I tried to move one but I honestly can't remember how I did it.

Reply:
The easiest way for testing purposes is to use the rails console; you can do `article.update(namespace: 2)` or similar.

Reply:
Oh right, thanks for this! Lemme get to it again.